assembly/src/parsers/ast/tests.rs

```diff
@@ -156,9 +156,7 @@ fn test_ast_program_serde_control_flow() {
     let source = "\
     begin
         repeat.3
-            push.1
-            push.0
-            push.1
+            push.1.0.1
```

Comment:
I would keep at least one simple `push`. Also, would be great to add `push` with hex data (e.g., `push.0xabc234` or something like this).